### ISSUE TITLE
Do not keep __wane__root in memory

### ISSUE THREAD

**Issue by lazarljubenovic** [wane/wane#17]
It seems like it's being used only once. Not sure what were original plans but it looks like it's not needed now.

So, instead of this...

```js
        this.__wane__root = document.body;
        this.__wane__data = new HelloWorldCmp();
        // Create a flat array of all DOM nodes which this component controls:
        this.__wane__domNodes = [
            /*0*/ __wane__createTextNode(this.__wane__data.message),
        ];
        // Creating the DOM tree:
        __wane__appendChildren(this.__wane__root, [
            this.__wane__domNodes[0],
        ]);
```

...we can have this.

```js
        this.__wane__data = new HelloWorldCmp();
        // Create a flat array of all DOM nodes which this component controls:
        this.__wane__domNodes = [
            /*0*/ __wane__createTextNode(this.__wane__data.message),
        ];
        // Creating the DOM tree:
        __wane__appendChildren(document.body, [
            this.__wane__domNodes[0],
        ]);
```

**Comment by lazarljubenovic** [comment]
It's used in `destroy` methods. We could pass it in there as well, but we can work on that only after destroy methods are optimized and no repeated over a bunch of times.
